rir/src/compiler/pir/type.h

```diff
@@ -242,7 +247,7 @@ struct PirType {
         return t;
     }
 
-    PirType forced() const {
+    PirType nonLazy() const {
```

Comment:
that's a terrible name! your `nonLazy` function removes the `lazy` flag and the `promiseWrapped` flag. In all the rest of the pir we call a `nonLazy` or `eager`, a value that is evaluated but maybe still promise wrapped.  how about `asValue`